src/pages/Detail/components/DetailAside/DetailAside.jsx

```diff
@@ -1,43 +1,40 @@
-import React, { useState } from 'react';
-import { AiOutlineHeart, AiFillHeart } from 'react-icons/ai';
+import React from 'react';
+import { useToggle } from 'pages/Detail/hooks/UseToggle';
 import { HiOutlineDotsVertical } from 'react-icons/hi';
-import { BsBookmark, BsFillBookmarkFill } from 'react-icons/bs';
 
 import DetailButton from '../DetailButtons/DetailButton';
 import DetailProfile from '../DetailProfile/DetailProfile';
+import HeartToggleButton from './HeartToggleButton';
+import BookMarkToggleButton from './BookMarkToggleButton';
 
-import styled, { css, keyframes } from 'styled-components';
+import styled, { keyframes, css } from 'styled-components';
 
 export default function DetailAside() {
-  const [isClickedIcon, setIsClickedIcon] = useState(false);
+  const { state: isClickedLike, handleToggle: setIsClickedLike } =
+    useToggle(false);
+  const { state: isClickedBookMark, handleToggle: setIsClickedBookMark } =
+    useToggle(false);
```

Comment:
```suggestion
  const [isClickedLike, setIsClickedLike] = useToggle(false);
  const [isClickedBookMark, setIsClickedBookMark] = useToggle(false);
```
useToggle이 배열을 리턴하게 하면 좀 더 편하게 네이밍 할 수 있겠네요

Reply:
네! 리뷰사항 반영하였습니다!